app/models/cmr.rb

```diff
@@ -369,8 +369,10 @@ def self.get_raw_granule_results(concept_id)
   # returns true if a granule with the specified concept id exists in cmr
   def self.raw_granule?(concept_id)
     url = Cmr.api_url('granules', 'echo10', 'concept_id': concept_id)
-    granule_xml = Cmr.cmr_request(url).parsed_response
-    granule_results = convert_to_hash('echo10', granule_xml)['results']
+    response = Cmr.cmr_request(url)
+    return false if response.code != 200
```

Comment:
would it be helpful to log anything here?

Reply:
I added a log message.